templates/common/.github/workflows/bicep/azure-dev.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
       AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
       AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+      AZURE_RESOURCE_GROUP: ${{ secrets.AZURE_RESOURCE_GROUP }}
+      AZURE_DEV_USER_AGENT: ${{ secrets.AZURE_DEV_USER_AGENT }}
```

Comment:
When we chatted, I was thinking that `AZURE_DEV_USER_AGENT` would be set in a template managed by starshot and not azd. We might need to choose an alternative approach here.

Reply:
Unfortunately looks like this file is common across templates and there isn't a way to specify this per template. Let me know if there's a better way. Also, is there any harm in keeping the "AZURE_DEV_USER_AGENT" as env variable here?